integration_test/third_party_apps_test/applications/couchdb/centos_rhel/install

```diff
@@ -30,6 +30,13 @@ esac
 
 sudo yum install -y couchdb
 
+# TODO(b/370026125): logs don't go to couchdb.log on RL8 as of 3.4.1.
+# Temporarily force logs to couchdb.log until that's resolved.
+# See https://github.com/apache/couchdb/issues/5261.
+sudo mkdir -p /var/log/couchdb
+sudo touch /var/log/couchdb/couchdb.log
+sudo chown couchdb:couchdb /var/log/couchdb/couchdb.log
```

Comment:
Since this is deliberate breakage, let's update `supported_app_version` in `couchdb/metadata.yaml` to say "up to 3.3.3" or something? And maybe add a note that logs go to syslog with 3.4.x (we might need a separate test directory for that)?

Reply:
Echoing from chat; I'm not a fan of indicating no support for 3.4.x when it works fine on DEB systems and the workaround for RPM systems is rather minor. Added a clarifying note in the docs instead.

In the interest of unblocking the release I'd like to defer the separate test directory to another PR.